modules/ipi-install-configuring-networking.adoc

```diff
@@ -42,7 +48,12 @@ $ sudo nohup bash -c "
 +
 [NOTE]
 ====
-The ssh connection might disconnect after executing these steps.
+If your bare-metal NIC connects to a switch, verify that no Bridge Data Unit Protocol (BPDU) guard exists on the switch interface, or disable Spanning Tree Protocol (STP) on the bare-metal bridge by appending the `bridge.stp no` argument to the `nmcli connection` command:
+
```

Comment:
The instructions are a little vague right now. Are the below statements correct?

If a there is a BPDU guard on the switch interface, you must disable STP by appending `bridge.stp no`. 
If there is no BPDU guard, you don't need to take any further action.